request_logging/middleware.py

```diff
@@ -11,27 +11,32 @@ class LoggingMiddleware(MiddlewareMixin):
 
     def process_request(self, request):
         request_logger.info(colorize("{} {}".format(request.method, request.get_full_path()), fg="cyan"))
-        if (request.body):
-            self.log_body(self.chunked_to_max(request.body))
+        headers = {k: v for k, v in request.META.items() if k.startswith('HTTP')}
```

Comment:
according to [this SO response](http://stackoverflow.com/questions/3889769/how-can-i-get-all-the-request-headers-in-django), it should be `k.startsWith('HTTP_')`. 
Also can you strip out the extra `HTTP_` just to avoid confusion and save some space?